Gemfile

```diff
@@ -1,4 +1,5 @@
 source "https://rubygems.org"
 
-gem "datura", git: "https://github.com/CDRH/datura.git", tag: "v0.2.0-beta"
+gem "datura", git: "https://github.com/CDRH/datura.git", tag: "v0.1.7"
 
+gem "byebug"
```

Comment:
Byebug should be dropped from this specific repo's Gemfile as it is now a dependency of Datura itself and will always be present for collection repos.

Reply:
removed